src/rules/__tests__/createEvaluateConditions.test.js

```diff
@@ -37,114 +37,99 @@ describe('createEvaluateConditions returns a function that when called', functio
       return result;
     };
 
-    var logConditionNotMet = emptyFn;
-    var logConditionError = emptyFn;
+    var logConditionNotMetSpy = jasmine.createSpy();
+    var logConditionErrorSpy = jasmine.createSpy();
+    var result = createEvaluateConditions(
+      executeDelegateModule,
+      isConditionMet,
+      logConditionNotMetSpy,
+      logConditionErrorSpy
+    )(rule);
 
-    expect(
-      createEvaluateConditions(
-        executeDelegateModule,
-        isConditionMet,
-        logConditionNotMet,
-        logConditionError
-      )(rule)
-    ).toBeTrue();
+    expect(result).toBeTrue();
+    expect(logConditionNotMetSpy).not.toHaveBeenCalled();
+    expect(logConditionErrorSpy).not.toHaveBeenCalled();
   });
 
-  it('returns false when conditions are not met', function () {
-    var executeDelegateModule = function (condition) {
-      return condition.modulePath === 'condition1';
-    };
-    var isConditionMet = function () {
-      return false;
-    };
-
-    var logConditionNotMet = emptyFn;
-    var logConditionError = emptyFn;
+  it('handles a condition returning a promise', function () {
```

Comment:
This test case is the result of merging two test cases that were duplicating the same scenario but that had two different assertions.